shared/README.md

```
@@ -0,0 +1,14 @@
+Files in `shared/` may be imported by both:
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)